stubs/twisted/web/client.pyi

```diff
@@ -1,8 +1,9 @@
-from typing import BinaryIO, Optional, Type, TypeVar
+from typing import BinaryIO, Optional, Sequence, Type, TypeVar
```

Comment:
We should probably include license headers in the stubs files?

Reply:
Hmm---possibly? They feel more like metadata derived from `twisted` rather than original creation of ourselves. Not sure what the right thing to do from a licensing perspective is there.